src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,46 @@
+.stars {
+  width: 166px;
+  height: 16px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 16px 16px 0;
+
+  &__container {
+    display: flex;
+  }
+
+  &__star::before {
+    content: '';
+    display: flex;
+    width: 16px;
+    height: 16px;
+    background-image: url('../images/star.svg');
+    background-size: 16px 16px;
+    background-repeat: no-repeat;
+    background-position: center;
+  }
+
+  &__star:nth-child(-n + #{$active-stars})::before {
```

Comment:
The use of '#{$active-stars}' in the nth-child selector suggests that a Sass variable is being used. Ensure that this variable is defined and passed correctly in the context where this CSS is used.